lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1223,6 +1273,57 @@ def _url(self, unit_number) -> str:
             self.port,
         )
 
+    def _check_alert_rules(self) -> bool:
+        """Check alert rules using Loki API.
+
+        Returns: bool
+
+        Emits:
+            loki_push_api_alert_rules_changed: This event is emitted when alert rules are checked.
+        """
+        url = "http://127.0.0.1:{}/loki/api/v1/rules".format(self.port)
+        req = request.Request(url)
+        try:
+            request.urlopen(req)
+        except HTTPError as e:
+            msg = e.read().decode("utf-8")
+
+            if e.code == 404 and "no rule groups found" in msg:
```

Comment:
Should we open an Issue on Loki asking to turn this into proper json?
And mention that issue in a comment here?

Reply:
I'm going to ask in the community chat first